lib/pipes/parseBoolean.pipe.ts

```diff
@@ -1,5 +1,13 @@
+import { BadRequestException } from '../exceptions';
+
 export function ParseBooleanPipe(value: any): boolean {
-  return typeof value === 'string' && (value === 'true' || value === 'false')
-    ? value.toLowerCase().trim() === 'true'
-    : value;
+  if (value === true || value === 'true') {
+    return true;
+  }
+
+  if (value === false || value === 'false') {
```

Comment:
Consideration, do we also want to allow `0` and `'0'`

Reply:
I'd say no.